pdfmaker/kassia.py

```diff
@@ -239,6 +247,14 @@ def linebreak(self,neumes,lyrics = None):
         #print neumePos
         return neumePos, lyricPos
 
+    def fill_page_dict(self, page_dict):
+        for attrib_name in page_dict:
+            if not page_dict.has_key(attrib_name):
+                page_dict[attrib_name] = 72
+            else:
+                page_dict[attrib_name] = int(page_dict[attrib_name])
+        return page_dict
+
```

Comment:
Yeah this wasn't the best (just quick and easy). I moved the default settings back up towards the top of createPDF().